changes.txt

```diff
@@ -0,0 +1,30 @@
+6d64cf67 Merge pull request #1265 from kubernetes-client/dependabot/github_actions/actions/setup-python-v2.1.3
```

Comment:
Is this file generated?

Reply:
Yeah. That upgrade was done by `dependabot`. It was enabled in https://github.com/kubernetes-client/python/pull/1259.